src/OgAccess.php

```diff
@@ -191,7 +191,8 @@ public static function userAccessEntity($operation, EntityInterface $entity, Acc
       foreach ($entity_groups as $groups) {
         foreach ($groups as $group) {
           $user_access = static::userAccess($group, $operation, $user);
-          if ($user_access->isAllowed()) {
+          if (!$user_access->isForbidden()) {
+            //this covers allowed and neutral
```

Comment:
This commit is from PR #133 - it crept in here accidentally. Best is to rebase the branch.